src/test/java/org/folio/rest/api/CirculationRulesApiTest.java

```diff
@@ -57,7 +57,7 @@ private JsonResponse getResponse() throws Exception {
   /** @return the JSON of the get response, asserts an HTTP_OK=200 response */
   private JsonObject get() throws Exception {
     JsonResponse response = getResponse();
-    assertThat(response.getBody(), response.getStatusCode(), is(HttpURLConnection.HTTP_OK));
+    assertThat(response.getBody(), response, matchesOk());
```

Comment:
```suggestion
    assertThat(response, matchesOk());
```
matchesOk = matchesSafely automatically appends response.getBody() to the failure message.